nevergrad/optimization/differentialevolution.py

```diff
@@ -71,7 +71,7 @@ class _DE(base.Optimizer):
 
     def __init__(self, instrumentation: Union[int, Instrumentation], budget: Optional[int] = None, num_workers: int = 1) -> None:
         super().__init__(instrumentation, budget=budget, num_workers=num_workers)
-        self._penalize_cheap_violations = False
+        self._penalize_cheap_violations = True
```

Comment:
isnt it the only algorithm which absolutely requires that the violations are penalized??

Reply:
argh my god sorry